pkg/kube/container_oci.go

```diff
@@ -264,27 +266,33 @@ func (t *containerTranslator) configureProcess() {
 	for _, gid := range security.GetSupplementalGroups() {
 		t.g.AddProcessAdditionalGid(uint32(gid))
 	}
-	if security.GetPrivileged() {
-		t.g.SetupPrivileged(true)
-	} else {
-		aaProfile := security.GetApparmorProfile()
-		aaProfile = strings.TrimPrefix(aaProfile, appArmorLocalhostPrefix)
-		t.g.SetProcessApparmorProfile(aaProfile)
-		for _, capb := range security.GetCapabilities().GetDropCapabilities() {
-			t.g.DropProcessCapabilityEffective(capb)
-			t.g.DropProcessCapabilityAmbient(capb)
-			t.g.DropProcessCapabilityBounding(capb)
-			t.g.DropProcessCapabilityInheritable(capb)
-			t.g.DropProcessCapabilityPermitted(capb)
-		}
-		for _, capb := range security.GetCapabilities().GetAddCapabilities() {
-			t.g.AddProcessCapabilityEffective(capb)
-			t.g.AddProcessCapabilityAmbient(capb)
-			t.g.AddProcessCapabilityBounding(capb)
-			t.g.AddProcessCapabilityInheritable(capb)
-			t.g.AddProcessCapabilityPermitted(capb)
-		}
+
+	aaProfile := security.GetApparmorProfile()
+	aaProfile = strings.TrimPrefix(aaProfile, appArmorLocalhostPrefix)
+	t.g.SetProcessApparmorProfile(aaProfile)
+
+	if err := setupSELinux(&t.g, security.GetSelinuxOptions()); err != nil {
```

Comment:
Maybe add a check to verify if both apparmor/selinux profiles are specified and return error in this case ?

Reply:
Nice idea, will add